kernel/kselftest.py.data/ftrace.yaml

```diff
@@ -0,0 +1,9 @@
+component: !mux
+    ftrace:
+        comp: 'ftrace'
+
+run_type: !mux
+    upstream:
+        type: 'upstream'
+        location: 'git://git.kernel.org/pub/scm/linux/kernel/git/powerpc/linux.git'
+        branch: 'merge'
```

Comment:
As you wish to be using a powerpc specific kernel as a default location here, could you please rename this file to `ftrace_powerpc.yaml` to keep things generic.
Rest LGTM, and thank you for the patch!!